code/modules/surgery/tools.dm

```diff
@@ -265,13 +265,13 @@
 	playsound(get_turf(user), 'sound/machines/click.ogg', 50, TRUE)
 	if(tool_behaviour == TOOL_SCALPEL)
 		tool_behaviour = TOOL_SAW
-		balloon_alert(user, ("[src] can now cut bones."))
+		balloon_alert(user, "energy increased")
```

Comment:
```suggestion
		balloon_alert(user, "enabled bone-cutting mode")
```

While yours is shorter, I think it's important to make it clear that the laser scalpel acts as a bonesaw (but also is at that point not really good as a scalpel).